modules/otel-config-send-metrics-monitoring-stack.adoc

```diff
@@ -38,7 +33,15 @@ spec:
           receivers: [otlp]
           exporters: [prometheus]
 ----
-<1> Configures the Operator to create the Prometheus `ServiceMonitor` CR to scrape the Collector's internal metrics endpoint and Prometheus exporter metric endpoints. The metrics will be stored in the OpenShift monitoring stack.
+<1> Configures the {OTELOperator} to create the Prometheus `ServiceMonitor` CR or `PodMonitor` CR to scrape the Collector's internal metrics endpoint and Prometheus exporter metrics endpoints.
+
+[NOTE]
+====
+Setting `enableMetrics` to `true` creates two `ServiceMonitor` instances:
```

Comment:
```suggestion
Setting `enableMetrics` to `true` creates the following two `ServiceMonitor` instances:
```

Reply:
instances or endpoints? Sticking to one term would help keep the terminology consistent.

Reply:
Instances